tests/conftest.py

```diff
@@ -16,41 +19,14 @@ def auth_fc(auth_fc_module):
 
 @pytest.fixture
 def auth_fc_module(request):
-    import sirepo.srunit
-    from pykern.pkcollections import PKDict
+    with _auth_client_module(request) as c:
+        yield c
 
-    cfg = PKDict(
-        SIREPO_AUTH_BASIC_PASSWORD='pass',
-        SIREPO_AUTH_BASIC_UID='dev-no-validate',
-        SIREPO_AUTH_EMAIL_FROM_EMAIL='x',
-        SIREPO_AUTH_EMAIL_FROM_NAME='x',
-        SIREPO_AUTH_EMAIL_SMTP_PASSWORD='x',
-        SIREPO_AUTH_EMAIL_SMTP_SERVER='dev',
-        SIREPO_AUTH_EMAIL_SMTP_USER='x',
-        SIREPO_AUTH_GITHUB_CALLBACK_URI='/uri',
-        SIREPO_AUTH_GITHUB_KEY='key',
-        SIREPO_AUTH_GITHUB_SECRET='secret',
-        SIREPO_AUTH_GUEST_EXPIRY_DAYS='1',
-        SIREPO_AUTH_METHODS='basic:email:guest',
-        SIREPO_FEATURE_CONFIG_API_MODULES='status',
-    )
-    if 'email3_test' in str(request.fspath.purebasename):
-        cfg.SIREPO_AUTH_METHODS += ':github'
-    else:
-        cfg.SIREPO_AUTH_DEPRECATED_METHODS = 'github'
-    from pykern import pkconfig
-    pkconfig.reset_state_for_testing(cfg)
 
-    from pykern import pkunit
-    from pykern import pkio
-    cfg['SIREPO_SRDB_ROOT'] = str(pkio.mkdir_parent(pkunit.work_dir().join('db')))
-    p, fc = _job_supervisor_start(request, cfg=cfg)
-    if p:
-        yield fc
-        p.terminate()
-        p.wait()
-    else:
-        yield sirepo.srunit.flask_client(cfg=cfg)
+@pytest.fixture
+def auth_uc_module(request):
```

Comment:
`uc` is too short. `fc` was used everywhere as a variable. Here, we are going to use this for uwsgi only. Therefore, I would call it uwsgi_module. The `auth` part is unnecessary to identify.

Reply:
https://github.com/radiasoft/sirepo/commit/66da4701448010669a6cbfea717df20863acec62